charts/featureform/charts/api-server/templates/deployment.yaml

```diff
@@ -3,38 +3,38 @@ kind: Deployment
 metadata:
   creationTimestamp: null
   labels:
-    run: sandbox-api-server
-  name: sandbox-api-server
+    run: featureform-api-server
+  name: featureform-api-server
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: sandbox-api-server
+      app: featureform-api-server
   strategy: {}
   template:
     metadata:
       creationTimestamp: null
       labels:
-        app: sandbox-api-server
+        app: featureform-api-server
     spec:
       containers:
         - image: {{ .Values.image.repository }}
-          name: sandbox-api-server
+          name: featureform-api-server
           ports:
             - containerPort: 7878
               protocol: TCP
           resources: {}
           env:
-            - name: TYPESENSE_PORT
-              value: "80"
-            - name: TYPESENSE_HOST
-              value: sandbox-typesense
-            - name: TYPESENSE_APIKEY
-              value: password
-            - name: ETCD_HOST
-              value: sandbox-etcd
-            - name: ETCD_PORT
-              value: "2379"
+            - name: API_PORT
+              value: "7878"
```

Comment:
Moved them into the sub-value files. Needs more thought to get them into the top level value file since I would like the subcharts to be run individually with an override from the top chart instead